dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerTest.java

```diff
@@ -235,7 +238,7 @@ public void testServlet31() throws Exception {
      *                       if something goes horribly wrong
      */
     @Test
-    @SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
+    @SkipForRepeat({NO_MODIFICATION, EE9_FEATURES})
```

Comment:
We can probably remove this since we have a test here for servlet 4.0: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServerTest.java#L106 and just ignore the 31 test for 4.0 and 5.0?